### PR TITLE
seapath_demo_cluster_definition: add basic template to make a cluster

### PR DESCRIPTION
**Pull request by TanguyRaufflet**
Add inventory based on the example template. But simplified, to create a very basic cluster for POC.
Refactor of the README file in the inventory's directory to provide more details about the inventories and their purpose.

### CODE REVIEW

File: inventories/README.md [hunk 1]
```diff
@@ -28,3 +48,35 @@ remove_all_networkd_config: true
 # If set to true, the network playbook will start by wiping the /etc/systemd/network/ directory content, this can help cleaning old conflicting files.
 # THIS MUST NOT BE USED WITH skip_recreate_team0_config at the same time or the cluster network config won't be recreated.
 ```
+
+### Demo SEAPATH cluster inventory
```
Comment (Paullgk) [inline inventories/README.md:52]: SEAPATH demo

File: inventories/README.md [hunk 2]
```diff
@@ -28,3 +48,35 @@ remove_all_networkd_config: true
...
+
+If you are SEAPATH newcomer, you can use this example to create a SEAPATH POC cluster. At the opposite of a real SEAPATH cluster, which requires an advance machine with multiple cores and network cards. This example only requires one network interface and can be made with costumer retail machines.
+
+Of course, to achieve this, some concessions have been made. We keep only one network, as described in the schema below. Some advanced features have been removed.
+
+<img src="./basic_cluster.png" alt="Exemple d'image" style="max-width:400px">
+
+#### ⚠  Demo SEAPATH cluster inventory unsupported features ⚠
```
Comment (Paullgk) [inline inventories/README.md:60]: SEAPATH demo

**Review by Paullgk — CHANGES_REQUESTED** [review]

File: inventories/README.md (hunk 2 above)
Comment (TanguyRaufflet) [reply inventories/README.md]: Done

File: inventories/README.md (hunk 1 above)
Comment (TanguyRaufflet) [reply inventories/README.md]: Done

File: inventories/README.md [hunk 3]
```diff
@@ -28,3 +48,35 @@ remove_all_networkd_config: true
...
+
+- Low latency network
+- Dedicated cluster network with redundancy
+- Advanced features like:
+    - SNMP
+    - Syslog log sending
+    - Advanced isolation
+
+
+#### Variables to be configured to use the demo SEAPATH :
```
Comment (Paullgk) [inline inventories/README.md:70]: SEAPATH demo

**Review by Paullgk — CHANGES_REQUESTED** [review]

**Review by Paullgk — APPROVED** [review]

**Pull request merged**